.ci/mingw.sh

```diff
@@ -0,0 +1,25 @@
+. "$CI_SCRIPTS/common.sh"
+
+# FIXME: When Travis gets a recent version of Mingw-w64 use this
+#sudo apt-get install binutils-mingw-w64-i686 gcc-mingw-w64-i686 g++-mingw-w64-i686 mingw-w64-dev mingw-w64-tools 
+sudo apt-get install wine 
+sudo apt-get install libc6-dev-i386 
+
+# mingw-w64 build from http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/rubenvb/gcc-4.8-release/
+wget "http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win32/Personal%20Builds/rubenvb/gcc-4.8-release/i686-w64-mingw32-gcc-4.8.0-linux64_rubenvb.tar.xz?r=http%3A%2F%2Fsourceforge.net%2Fprojects%2Fmingw-w64%2Ffiles%2FToolchains%2520targetting%2520Win32%2FPersonal%2520Builds%2Frubenvb%2Fgcc-4.8-release%2F&ts=1422959985&use_mirror=heanet" -O mingw.tar.xz
```

Comment:
It can be shortened to:

    wget http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win32/Personal%20Builds/rubenvb/gcc-4.8-release/i686-w64-mingw32-gcc-4.8.0-linux64_rubenvb.tar.xz -O mingw.tar.xz

OpenWRT uses urls similar to the one I specified above.